frontend/encore/dev-server.rst

```diff
@@ -113,3 +116,4 @@ your page. HMR works automatically with CSS (as long as you're using the
     CSS. That is no longer needed.
 
 .. _`webpack-dev-server`: https://webpack.js.org/configuration/dev-server/
+.. _`it's not recommended to disable the firewall`: https://webpack.js.org/configuration/dev-server/#devserverdisablehostcheck
```

Comment:
This is a little odd because this links to a different option.. the problem is that dev-server v4 still is in beta, and its docs don't exist. The disable-host-check v3 === firewall v4 (is my understanding)